src/amo/components/SearchResult.js

```diff
@@ -16,16 +19,24 @@ export default class SearchResult extends React.Component {
         <Link to={`/${lang}/firefox/addon/${result.slug}/`}
               className="SearchResult-link"
               ref={(el) => { this.name = el; }}>
-          <div>
-            <img className="SearchResult-icon" src={result.icon_url}
-                 alt="Icon" />
-          </div>
           <section className="SearchResult-main">
+            <img className="SearchResult-icon" src={result.icon_url}
+                 alt="" />
             <h2 className="SearchResult-heading">
               <span className="SearchResult-name">
                 {result.name}
               </span>
             </h2>
+            <ul className="SearchResult-authors">
+              {result.authors.map((author) => (
+                <li className="SearchResult-author">{author.name}</li>
+              ))}
+            </ul>
+            <div className="SearchResult-users">
+              {sprintf(_('%(users)s users'), {
```

Comment:
You should actually grab `sprintf` and `gettext` from props. You'll need to wrap your component in with `translate()` like in [AddonDetail](https://github.com/mozilla/addons-frontend/blob/574d5410f457b2ab66f39138fece2c44010b9df0/src/amo/components/AddonDetail.js#L109) and then you can grab `const { i18n } = this.props;` and use `i18n.sprintf()` and `i18n.gettext()`.

The `core/utils:gettext()` was just a basic wrapper to mark the strings we'll need to translate before we had the real setup done. It should really go away now.